pytest/tests/stress/perf_ft_transfer.py

```diff
@@ -0,0 +1,471 @@
+            try:
+                tx = tx_queue.get_nowait()
+            except queue.Empty:
+                break
+            # Send out the transaction immediately.
+            try:
+                tx.poll(node, block_hash)
+            except (requests.exceptions.ReadTimeout, requests.exceptions.ConnectionError):
+                pass
+            my_transactions.put(tx)
+
+        try:
+            tx = my_transactions.get_nowait()
+        except queue.Empty:
+            time.sleep(0.1)
+            continue
+        try:
+            poll_result = tx.poll(node, block_hash)
+        except (requests.exceptions.ReadTimeout, requests.exceptions.ConnectionError):
+            my_transactions.put(tx)
+            continue
+        if not poll_result:
+            my_transactions.put(tx)
+            if tx.ttl != DEFAULT_TRANSACTION_TTL_SECONDS:
+                time.sleep(0.1) # don't spam RPC too hard...
+        else:
+            tx_queue.complete()
+
+def main():
+    parser = argparse.ArgumentParser(description='FT transfer benchmark.')
+    parser.add_argument('--fungible-token-wasm', required=True,
+        help='Path to the compiled Fungible Token contract')
+    parser.add_argument('--setup-cluster', default=False,
+        help='Whether to start a dedicated cluster instead of connecting to an existing local node',
+        action='store_true')
+    parser.add_argument('--contracts', default='0,2,4,6,8,a,c,e',
+        help='Number of contract accounts, or alternatively list of subnames, separated by commas')
+    parser.add_argument('--contract-key', default=None,
+        help='Account to deploy contract to and use as source of NEAR for account creation')
+    parser.add_argument('--accounts', default=1000, help='Number of accounts to use')
+    parser.add_argument('--no-account-topup', default=False,
+        action='store_true', help='Fill accounts with additional NEAR prior to testing')
+    parser.add_argument('--shards', default=10, help='number of shards')
+    parser.add_argument('--executors', default=2, help='number of transaction executors')
+    args = parser.parse_args()
+
+    logger.warning(f"SEED is {SEED}")
+    rng = random.Random(SEED)
+
+    if args.setup_cluster:
+        config = cluster.load_config()
+        nodes = cluster.start_cluster(2, 0, args.shards, config, [["epoch_length", 100]], {
+            shard: { "tracked_shards": list(range(args.shards)) }
+            for shard in range(args.shards + 1)
+        })
+        if args.contract_key is None:
+            signer_key = nodes[0].signer_key
+        else:
+            signer_key = key.Key.from_json_file(args.contract_key)
+
+    else:
+        nodes = [
+            cluster.RpcNode("127.0.0.1", 3030),
+        ]
+        # The `nearup` localnet setup stores the keys in this directory.
+        if args.contract_key is None:
+            key_path = (pathlib.Path.home() / ".near/localnet/node0/shard0_key.json").resolve()
```

Comment:
also, was this test script meant to be connected to the other node setup script? Doesn't seem to be possible to use it with that as it is now unless I'm missing something

Reply:
This is something @akashin added, I then added the `--contract-key` argument later. I think we can remove this entirely and use `--contract-key` exclusively.